saga-demo/saga-dubbo-demo/README.md

```diff
@@ -15,10 +15,88 @@ You will need:
 [alpha_server]: https://github.com/apache/incubator-servicecomb-saga/tree/master/alpha
 
 ## Running Demo
-```bash
-$ mvn clean install -DskipTests -Pdocker
-$ ./saga-dubbo-demo.sh up
+1. run the following command to create docker images in saga project root folder.
+   ```
+   mvn clean install -DskipTests -Pdocker -Pdemo
+   ```
+2. Enter the saga dubbo demo directory and give permissions to script
+   ```
+   cd ./saga-demo/saga-dubbo-demo
+   chmod +x saga-dubbo-demo.sh
+   ```
+3. start the whole application up(including alpha server and three demo services)
+   ```
+   ./saga-dubbo-demo.sh up
+   ```
+    **Note:** If you prefer to use MySQL as alpha's backend database, you need to try the following steps instead:
+    1. add dependency of `mysql-connector-java` in `alpha/alpha-server/pom.xml`
+     ```xml
+         <dependency>
+           <groupId>mysql</groupId>
+           <artifactId>mysql-connector-java</artifactId>
+         </dependency>
+     ```
+    2. remove alpha server's docker image
```

Comment:
The alpha server docker image could be overridden when we regenerate the docker image. 
So we don't need to remove the alpha server docker image here.

Reply:
i will change it